app/views.py

```diff
@@ -7,18 +7,22 @@
 
 def register(app):
 
-    @app.route("/data/")
-    def data():
-        return render_template("data.html")
-
     @app.route("/")
     def index():
         return render_template("index.html")
```

Comment:
Is it intentional that this displays the agencies page?

Reply:
Yes, based on the wireframe the index page should land users on the agencies list